src/infra/alarm/template.yaml

```
@@ -97,3 +97,318 @@ Resources:
+      Statistic: Sum
+      Threshold: 5
+      EvaluationPeriods: 1
+      Period: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: service
+          Value: InterventionsProcessorFunction
+
+  LevelOfConfidenceTooLow:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: "Level of confidence is too low."
+      Namespace: !Ref SystemTagValue
+      MetricName: CONFIDENCE_LEVEL_TOO_LOW
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Statistic: Sum
+      Threshold: 5
+      EvaluationPeriods: 1
+      Period: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: service
+          Value: InterventionsProcessorFunction
+
+  InterventionEventStale:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: "Intervention event messages are out of order."
+      Namespace: !Ref SystemTagValue
+      MetricName: INTERVENTION_EVENT_STALE
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Statistic: Sum
+      Threshold: 5
+      EvaluationPeriods: 1
+      Period: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: service
+          Value: InterventionsProcessorFunction
+
+  TooManyInterventionEventStale:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSHighAlertNotificationTopicARN
+      AlarmDescription: "Too many intervention event messages are out of order."
+      Namespace: !Ref SystemTagValue
+      MetricName: INTERVENTION_EVENT_STALE
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Statistic: Sum
+      Threshold: 5
+      EvaluationPeriods: 5
+      Period: 60
+      TreatMissingData: notBreaching
+      Dimensions:
+        - Name: service
+          Value: InterventionsProcessorFunction
+
+  DBQueryNoResponseError:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSLowAlertNotificationTopicARN
+      AlarmDescription: "DynamoDB query no response error"
+      ComparisonOperator: GreaterThanOrEqualToThreshold
+      Threshold: 1
+      EvaluationPeriods: 1
+      TreatMissingData: notBreaching
+      MetricName: DB_QUERY_ERROR_NO_RESPONSE
+      Namespace: !Ref SystemTagValue
+      Dimensions:
+        - Name: service
+          Value: InterventionsProcessorFunction
+      Period: 60
+      Statistic: Sum
+      Unit: Count
+
+  TooManyDBQueryNoResponseError:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSHighAlertNotificationTopicARN
+      AlarmDescription: "DynamoDB query no response error five times within a minute."
```

Bit of a nit pick here, sorry for being a nightmare, as a general comment on this file, could the alarm description field be reviewed to improve readability and have a more standardised format (full stops at the end etc)
```suggestion
      AlarmDescription: "The query to DynamoDB has provided the no response error five times within a minute."
```